tailscale/DOCS.md

```diff
@@ -203,6 +203,24 @@ This option lets you to specify a custom control server instead of the default
 (`https://controlplane.tailscale.com`). This is useful if you are running your
 own Tailscale control server, for example, a self-hosted [Headscale] instance.
 
+### Option: `map4via6`
+
+This option enables [4via6 routing][tailscale_info_4via6]. All subnets in 
+**advertise_routes** will use it, even if it is populated automatically by this 
```

Comment:
Other parts of the DOCS reference options sg. like this:
All subnets in `advertise_routes` option will use it...